content/en/blog/Kubernetes-basics/index.md

```diff
@@ -0,0 +1,207 @@
+---
+title: "Kubernetes Basics"
+description: "Introduction to Kubernetes"
+excerpt: "Explore various strategies and best practices for managing branches in Git"
+date: 2024-10-16T12:06:50-04:00
+lastmod: 2024-10-16T12:06:50-04:00
+draft: false
+weight: 50
```

Comment:
can you change weight: 43 to show it on the top.

Reply:
Done